src/evidently/collector/config.py

```diff
@@ -53,6 +53,22 @@ def is_ready(self, config: "CollectorConfig", storage: "CollectorStorage") -> bo
             self.last_triggered = now
             return True
         return False
+    
+
+class RowsCountTrigger(CollectorTrigger):
+    rows_count: int = 1
+
+    def is_ready(self, config: "CollectorConfig", storage: "CollectorStorage") -> bool:
+        print("---")
+        print("RowsCountTrigger")
+        print(f"rows_count: {self.rows_count}")
+        print(f"buffer length:{len(storage._buffers[config.id])}")
+        if len(storage._buffers[config.id]) > 0: 
+            if len(storage._buffers[config.id]) % self.rows_count == 0:
```

Comment:
Are you sure you want to use modulo here instead of just `>=`? It means that if rows_count is 10, report will trigger only when buffer size is 10, 20, 30 etc, but not 11, 15, 27 etc
Also can merge this check with previous one via `and`.

Reply:
Combined both conditions and used `>=` instead of `%` in a705857e636245a7bd2fec6edbac01975c45434d